jdaviz/configs/cubeviz/plugins/viewers.py

```diff
@@ -20,17 +20,19 @@ class CubevizImageView(BqplotImageView, JdavizViewerMixin):
     # ones are added in glue-jupyter in future that we don't want here.
     inherit_tools = False
 
+    # TODO: 'jdaviz:boxzoommatch' and 'jdaviz:panzoommatch' would be nice
+    # but they break spectrum collapse in Cubeviz if used as-is from Imviz.
```

Comment:
this shouldn't be too hard to fix (worth filing a follow-up ticket?), the match zoom tools were just written for imviz with the assumption that all viewers were of the same type (and could have the same reference data).

Reply:
I am not sure what was going on. The traceback was weird enough that I decided not to piggyback this feature on this PR. Might be something to do with linking and viewer limit assumptions too. Definitely need to be pointed at least 5 points and picked up as a proper ticket.

Reply:
p.s. But then again, POs not asking for it, so 🤷